RELEASE.md

```diff
@@ -6,4 +6,39 @@ No manual creation of a midstream `release-v1.x` branch is needed. The nightly J
 
 ## Enable CI for the release branch
```

Comment:
Would it be possible for serverless-qe robot or by some other way to create a PR against openshift/release when a new Serving release branch is ready? Also same should be done for net-* repos? @alanfx wdyth?